tools/ci_build/github/azure-pipelines/templates/win-ci.yml

```diff
@@ -37,13 +34,16 @@ parameters:
   type: boolean
   default: true
 
+- name: buildNodejs
```

Comment:
Do we need such a parameter? When it is false?

Reply:
when it is on x86 and armv7. the build will fail.

Reply:
Armv7? We don’t have such a build